scopes/component/component/ui/component-model/component-model.ts

```diff
@@ -29,6 +29,7 @@ export type ComponentModelProps = {
   host?: string;
   latest?: string;
   preview?: ComponentPreview;
+  snaps?: LegacyComponentLog[];
```

Comment:
once we change the other to logs, let's change it here (entire file) to refer to log or history

Reply:
yup